config/schedule.rb

```diff
@@ -4,11 +4,11 @@
 # http://en.wikipedia.org/wiki/Cron
 # Learn more: http://github.com/javan/whenever
 #
-set :output, 'log/schedule.log'
+# set :output, 'log/schedule.log'
```

Comment:
probably best to remove these if they aren't needed rather than comment them out 👍